test/unit/result_test.js

```diff
@@ -0,0 +1,49 @@
+const connOption = require("../integration/connectionOptions");
+const snowflake = require('snowflake-sdk');
+const assert = require('assert');
+const testUtil = require("../integration/testUtil");
+
+snowflake.configure({logLevel:"TRACE"});
+
+describe.only("test and log how many result.js called for each execution",function (){
+     let connection = snowflake.createConnection(connOption.valid);
+      connection.connect(
+        function (err, conn)
+        {
+          if (err)
+          {
+            console.error('Unable to connect: ' + err.message);
+          }
+          else
+          {
+            console.log('Successfully connected to Snowflake');
+          }
+        }
+      )
+    
+    it('Run Select 1 query', function (done)
+    {
+      connection.execute({
+        sqlText: "select 1",
+        complete: function (err, stmt)
+        {
+          testUtil.checkError(err);
+          var stream = stmt.streamRows();
```

Comment:
you are reading from stream here - so you need to set `streamResult ` to true as described in https://docs.snowflake.com/en/developer-guide/node-js/nodejs-driver-consume#streaming-results

Reply:
I set this because this is a testing code. Although I set streamResult:true option, the driver still called result.js twice. I remove stream and just simple check